src/CairoTTY.cc

```diff
@@ -115,6 +148,26 @@ void CairoTTY::NewPage()
     Home();
 }
 
+void CairoTTY::SetFontName(const std::string family)
```

Comment:
Just a minor thing: complex objects like a `std::string` should be passed by `const&` unless the callee needs to keep it (and then one could consider rvalue ref - `std::string&&`).

But this is not something that needs to be changed now. I will change that when I touch the sources in master.